tests/Form/InputTextTest.php

```
@@ -282,13 +438,25 @@ public function testRenderWithForm(): void
+            <input id="inputtext" type="text" form="form-id">
             HTML,
-            InputText::tag()->form('form-id')->id('inputtext-')->render(),
+            InputText::tag()->form('form-id')->id('inputtext')->render(),
             "Failed asserting that element renders correctly with 'form' attribute.",
         );
     }
 
+    public function testRenderWithGenerateId(): void
+    {
+        /** @phpstan-var string $id */
+        $id = InputText::tag()->getAttribute('id', '');
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)